src/v2/templates/people/person-list.php

```diff
@@ -158,16 +158,22 @@ function emptyOrUnassignedJSON($stuff): string
                             <?php
                     }
                     ?>
-                <td><?= $person->getId() ?></td>
-                <td><?= $person->getLastName()?></td>
-                <td><?= $person->getFirstName()?></td>
-                <td><?= $person->getAddress() ?></td>
-                <td><?= $person->getHomePhone() ?></td>
-                <td><?= $person->getCellPhone() ?></td>
-                <td><?= $person->getEmail() ?></td>
-                <td><?= emptyOrUnassigned($person->getGenderName()) ?></td>
-                <td><?= emptyOrUnassigned($person->getClassificationName()) ?></td>
-                <td><?= emptyOrUnassigned($person->getFamilyRoleName()) ?></td>
+
+                <?php
+                $columns = json_decode(SystemConfig::getValue('sPersonListColumns'), null, 512, JSON_THROW_ON_ERROR);
+                foreach ($columns as $columnKey => $columnValue) {
```

Comment:
if we are not using $columnKey, can't we simplify this?

```suggestion
                foreach ($columns as $columnValue) {
```

Reply:
done